source/sass/blocks/contacts-list.scss

```
@@ -0,0 +1,14 @@
+.contacts-list__link {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  text-transform: uppercase;
```

Модификаторы пишем через амперсанд 
```suggestion
  text-transform: uppercase;
  
  &--main {
  ...
```